app/controllers/concerns/turbo_streamable.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+module TurboStreamable
+  extend ActiveSupport::Concern
+
+  included do |controller|
+    around_action :handle_error, only: controller.respond_to?(:turbo_stream_actions) ? controller.turbo_stream_actions : []
+  end
+
+  def render_toast(message, success = true)
+    turbo_stream.append("toasts", partial: "shared/toast", locals: { message:, success: })
+  end
+
+  class_methods do
+    def render_toast = render_toast
+  end
```

Comment:
this does not do what you want it to: `controller` is the class itself, which will never respond to the instance method `turbo_stream_actions` (unless you define it as `self.turbo_stream_actions), so the error action will never be applied.

there's a much nicer way to achieve what you want, e.g.:

```suggestion
  class_methods do
    def turbo_stream_actions(*actions)
      around_action :handle_error, only: actions
    end
  end

  def render_toast(message, success = true)
    turbo_stream.append("toasts", partial: "shared/toast", locals: { message:, success: })
  end
```

with that you can include the module right at the beginning of the class, and then just call `turbo_stream_actions :create, :destroy` wherever you need it.